R/select_best.R

```diff
@@ -277,6 +281,20 @@ select_by_one_std_err.tune_results <- function(x, ..., metric = NULL) {
         .bound = bound
       ) %>%
       dplyr::filter(mean <= .bound)
+  } else if (direction == "zero") {
+    best_index <- which.min(abs(res$mean))
+    best <- res$mean[best_index]
+    bound_lower <- -abs(best) - res$std_err[best_index]
+    bound_upper <- abs(best) + res$std_err[best_index]
```

Comment:
A second set of eyes on these bound calculations would be appreciated—I believe this is the approach we want to take, though I'm not sure if we need to update any docs to clarify the definition of this bound.